buildspec.yml

```diff
@@ -0,0 +1,29 @@
+version: 0.2
+
+batch:
+  fast-fail: false
+  build-list:
+    - identifier: add_esdk_complete
+      buildspec: codebuild/add-esdk-complete.yml
+      env:
+        image: aws/codebuild/amazonlinux2-x86_64-standard:3.0
```

Comment:
Minor: if we generally want the same image I think we could just configure the codebuild project to use this image and only override when necessary.